design/FY2016/CondFDNodeFluxOutput.md

```diff
@@ -0,0 +1,120 @@
+Conduction Finite Difference Node Heat Flux Reporting
+================
+
+**M.J. Witte, GARD Analytics, Inc.**
+
+ - October 30, 2015
+ - *Revision Date*
+ 
+
+## Justification for New Feature ##
+
+The primary motivation for this work is to support experimental testing where heat flux meters are embedded within a wall. It is also useful for other work to show the flow of heat within material layers, especially for phase change material (PCM) applications.
+
+## E-mail and  Conference Call Conclusions ##
+
+*insert text*
+
+## Overview ##
+
+Currently, the conduction finite difference algorithm (HeatBalanceAlgorithm,ConductionFiniteDifference) reports the temperature at each node within a surface.  This will add reporting of the heat flux between nodes.
+
+## Approach ##
+
+At the end of the conduction finite difference (CondFD) solution for each timestep, the surface inside face conduction heat flux and the node temperatures will be used to calculate the heat flux between nodes.  The figure below illustrates the conduction finite difference node layout (from the Engineering Reference).
+
+
+![CondFD Nodes adapted from Engineering Reference](CondFDNodeFluxOutput-CondFDNodes-image176.png)
+
+Figure 1. Node depiction for Conduction Finite Difference Model (Nodes are numbered from outside to outside face to inside face.  External surface node = 1, Interior surface node = 8)
+
+For every CondFD surface, the Surface Inside Face Conduction Heat Transfer Rate per Area is available as an output. Starting with this flux, the heat flux from the Interior surface node 8 to the next interior node 7 is the Surface Inside Face Conduction Heat Transfer Rate per Area plus any change in heat storage in the material thickness associated with node 8.  Once the flux from node 8 to node 7 is determined, then the next flux can be calculated, and so on.
```

Comment:
I think this needs to be explicit about whether the heat flux is being calculated for the faces (interface between control volumes) or for right at the node (middle of control volume).  Node-to-node heat flux could be defined either way.  Because the material can change at the face, I assume you intend to report heat flux result for the internal faces.  The existing surface flux reports are helped by the half-node formulation so that the face and node are co-located.  I just want to caution that the situation is quite different for the full nodes and the actual equations will like differ from those used for the half nodes.